scripts/config.pl

```diff
@@ -26,6 +26,8 @@
 #   MBEDTLS_ECP_DP_M221_ENABLED
 #   MBEDTLS_ECP_DP_M383_ENABLED
 #   MBEDTLS_ECP_DP_M511_ENABLED
+#   MBEDTLS_MEMORY_BACKTRACE
+#   MBEDTLS_MEMORY_BUFFER_ALLOC_C
```

Comment:
This list hasn't been kept up-to-date, and it's useless since it's just a comment that duplicates information a few lines below. How about removing it?

Reply:
I prefer to update the list instead.